apps/pie-docs/src/components/text-input/overview/overview.md

```diff
@@ -0,0 +1,492 @@
+
+{% contentPageImage {
+    src:"../../../assets/img/components/text-input/behaviour-assistivetext.svg",
+    alt: "Example of assistive text behavior.",
+    width: "300px"
+} %}
+
+### Payment method
+
+{% notification {
+  type: "warning",
+  message: "This is the suggested pattern for development, this functionality, if required is handled by pillar engineers'."
+} %}
+
+When using the payment method, the specific card icon will appear as soon as the system detects the type of card entered by the user.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/text-input/behaviour-paymentmethod.svg",
+    alt: "Example of payment method behavior.",
+    width: "884px"
+} %}
+
+---
+
+## Interactive states
+
+Outlines the atomic level interactive elements for the component.
+
+{% contentLayout %}
+  {% contentItem %}
+    <h4>Default</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/text-input/interactivestates-default.svg",
+      width: "256px",
+      alt: "Default interactive state example."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Hover</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/text-input/interactivestates-hover.svg",
+      width: "256px",
+      alt: "Interactive state example when input is hovered over."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Focus</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/text-input/interactivestates-focus.svg",
+      width: "256px",
+      alt: "Interactive state example when input is focused."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Disabled</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/text-input/interactivestates-disabled.svg",
+      width: "256px",
+      alt: "Interactive state example when input is disabled.",
+      variant: "secondary"
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Read only</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/text-input/interactivestates-readonly.svg",
+      width: "256px",
+      alt: "Interactive state example in read only mode."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h4>Error</h4>
+    {% contentPageImage {
+      src: "../../../assets/img/components/text-input/interactivestates-error.svg",
+      width: "256px",
+      alt: "Interactive state example showing an error."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+---
+
+## Examples
+
+Outlines the atomic level interactive elements for the component.
+
+### LTR examples
+
+Here are some examples of Buttons in left-to-right context:
+
+{% contentPageImage {
+    src:"../../../assets/img/components/text-input/examples-LTR-default.svg",
+    alt: "Default example layout with left-to-right text alignment.",
```

Comment:
I think "direction" is more accurate. You can still have right alignment in left-to-right.
```suggestion
    alt: "Default example layout with left-to-right text direction.",
```